src/databricks/labs/ucx/source_code/known.json

```diff
@@ -25511,6 +25511,95 @@
   "spark-ocr": {
     "sparkocr": []
   },
+  "sparkdl": {
+    "sparkdl": [],
+    "sparkdl.estimators": [],
+    "sparkdl.estimators.keras_image_file_estimator": [
+      {
```

Comment:
@ericvergnaud : I saw you removing this in another PR. Should it be here?

Reply:
What has been removed is:
 - 'not computed' advices
 - test modules
So yes I believe it should be there